amundsen_application/static/css/_icons.scss

```diff
@@ -43,6 +49,7 @@ span.icon {
   @include iconBackgrounds($data-stores);
   @include iconBackgrounds($dashboards);
   @include iconBackgrounds($users);
+  @include iconBackgrounds($check);
 
```

Comment:
Could you please add a comment saying that the code below is deprecated and we shouldn't add more icons this way? 

I should have done it before! Thanks!